pkgs/development/python-modules/simplification/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools-scm,
+  numpy,
+  cython,
+  rdp,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "simplification";
+  version = "0.7.12";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "urschrei";
+    repo = "simplification";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-DgPzcpg6x7kkNfOGypbdVys5iYMyLlf+SjOE1Mr7A+A=";
+  };
+
+  prePatch = ''
+    cp ${rdp}/{include/header.h,lib/librdp.so} src/simplification
```

Comment:
Currently fails with
> cp: cannot stat '/nix/store/vsbz8y1y4bmwbnm457di7dni29kca0zd-rdp-0.13.0/lib/librdp.so': No such file or directory

Reply:
can you give details about this? I cannot reproducible

Reply:
Ah, now I remember. I copied it from the log of ofborg building for Darwin. You need to use dylib there. Try `${stdenv.hostPlatform.extensions.sharedLibrary}` instead of `.so`